include/definitions.h

```diff
@@ -90,8 +92,8 @@
 #define SPI_RST 21
 
 #define SPI_CS_TFT 36
-#define SPI_CS_POTI 33
-#define SPI_CS_SDCARD 14
+#define SPI_CS_POTI 0 //33
+#define SPI_CS_SDCARD 0 //14
 
```

Comment:
Again, does not look right go me.

Reply:
fixed: Poti uses I2C and SDCard has GPIO14